modules/tools/kube-prometheus-stack/main.tf

```diff
@@ -42,4 +42,79 @@ resource "helm_release" "prometheus_resources" {
   depends_on = [
     helm_release.kube_prometheus_stack,
   ]
-}
\ No newline at end of file
+}
+
+resource "kubernetes_manifest" "schedule_velero_prometheus_daily_backup" {
+  count = var.enable_velero ? 1 : 0
+  manifest = {
+    "apiVersion" = "velero.io/v1"
+    "kind"       = "Schedule"
+    "metadata" = {
+      "name"      = "prometheus-daily-backup"
+      "namespace" = "velero"
+    }
+    "spec" = {
+      "schedule" = "0 1 * * *"
+      "template" = {
+        "includedNamespaces" = [
+          "monitoring",
+        ]
+        "includedResources" = [
+          "*",
+        ]
+        "labelSelector" = {
+          "matchLabels" = {
+            "app" = "prometheus"
+          }
+        }
+        "snapshotVolumes" = true
+        "storageLocation" = "default"
+        "ttl"             = "72h0m0s"
+        "volumeSnapshotLocations" = [
+          "aws-s3",
+        ]
+      }
+    }
+  }
+  depends_on = [
+    helm_release.kube_prometheus_stack
+  ]
```

Comment:
is there a reason all the other schedule resources depend on velero_status except this one?